storage/scylladb_utils.h

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#ifndef TA_SCYLLADB_UTILS_H_
+#define TA_SCYLLADB_UTILS_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include <stdlib.h>
+#include "accelerator/errors.h"
+#include "cassandra.h"
+#include "utils/logger.h"
+
+logger_id_t scylladb_logger_id;
+
+/**
+ * @brief print error message of CassFuture
+ *
+ * @param[in] future result of an operation
+ * @return
+ * - none
+ */
+void print_error(CassFuture* future);
```

Comment:
I don't think it is worth to expose `print_error` here. Instead, just put it into C source as a static function.